src/components/v2/V2Create/tabs/FundingCycleTab/FundingCycleTabContent.tsx

```diff
@@ -80,9 +87,26 @@ export default function FundingCycleTabContent({
           }
           primaryColor={tokenDrawerSeen ? seenColor : undefined}
         />
+        {featureFlagEnabled('nftRewards') ? (
```

Comment:
**suggestion** a constant for `const isNFTRewardsEnabled = featureFlagEnabled('nftRewards')`, re-use it here and blow line 109?